.github/workflows/release.yml

```
@@ -575 +1,562 @@
+      - name: Build and Package
+        run: |
+          dart pub global activate melos 6.3.2
+          melos bootstrap
+          cd apps/flipper
+          flutter clean
+          dart run msix:create --store
+
+      - name: Extract msix_version
+        id: get_version
+        shell: powershell
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)